fenix/app/src/main/java/org/mozilla/fenix/debugsettings/ui/DebugDrawer.kt

```diff
@@ -6,31 +6,118 @@ package org.mozilla.fenix.debugsettings.ui
 
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material.Snackbar
+import androidx.compose.material.SnackbarHost
+import androidx.compose.material.SnackbarHostState
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.launch
+import mozilla.components.support.ktx.android.content.appName
+import mozilla.components.support.ktx.android.content.appVersionName
+import org.mozilla.fenix.compose.Divider
 import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.compose.inComposePreview
+import org.mozilla.fenix.compose.list.TextListItem
 import org.mozilla.fenix.theme.FirefoxTheme
 
 /**
  * The debug drawer UI.
  */
 @Composable
-fun DebugDrawer() {
-    Box(
+fun DebugDrawer(
+    navigationItems: List<String>,
+    onNavigationItemClick: (String) -> Unit,
```

Comment:
need a second opinion on these parameter names. The only other potential candidate I had was something along the lines of "debugDestinations", but navigation items seemed to fit the bill better.

Reply:
how about something like `menuItems`? Then that would read as Debug drawer menu items

Reply:
+1 `menuItems`. then there isn't an implied opinion about whether they need to actually navigate, or if they're top-level.

Reply:
Could also introduce a type here so we don't end up needing to do String lookups between the two. Something like:

`data class DebugDrawerMenuItem(val label: String, val onClick: () -> Unit)`

Reply:
I'm a fan of `menuItems`! I'm actually going to make a class or an enum for menu items as part of the [navigation middleware ticket](https://bugzilla.mozilla.org/show_bug.cgi?id=1864075), so we can either do the latter half-stop refactoring now or as part of the [navigation middleware integration](https://bugzilla.mozilla.org/show_bug.cgi?id=1864076).